gremlin-driver/src/main/java/org/apache/tinkerpop/gremlin/driver/Cluster.java

```diff
@@ -993,6 +1002,16 @@ public Builder connectionSetupTimeoutMillis(final long connectionSetupTimeoutMil
             return this;
         }
 
+        /**
+         * Configures whether cluster will send a user agent during
+         * web socket handshakes
+         * @param enableUserAgentOnConnect true enables the useragent. false disables the useragent.
+         */
+        public Builder enableUserAgentOnConnect(boolean enableUserAgentOnConnect) {
+            this.wsHandshakeUserAgentEnabled = enableUserAgentOnConnect;
```

Comment:
it seems like you went with the rename i suggested - please rename the underlying variables as well to keep things consistent.

nit: `final`